helix-term/src/commands/typed.rs

```diff
@@ -2507,7 +2508,19 @@ fn read(cx: &mut compositor::Context, args: &[Cow<str>], event: PromptEvent) ->
     ensure!(args.len() == 1, "only the file name is expected");
 
     let filename = args.first().unwrap();
-    let path = PathBuf::from(filename.to_string());
+    let path = PathBuf::from(if filename.starts_with('~') {
```

Comment:
There's a helper `helix_stdx::path::expand_tilde` that can be used here instead

Reply:
There is indeed, fixed o/